src/H5Pfcpl.c

```diff
@@ -1165,32 +1165,21 @@ H5Pget_shared_mesg_phase_change(hid_t plist_id, unsigned *max_list /*out*/, unsi
 } /* end H5Pget_shared_mesg_phase_change() */
 
 /*-------------------------------------------------------------------------
- * Function:	H5Pset_file_space_strategy
+ * Function:	H5P__set_file_space_strategy
```

Comment:
Made package routines for these calls, to support H5Pget/set_file_space changes.